src/observer.ts

```diff
@@ -49,21 +49,21 @@ export class Observer {
    * completion of the task.
    * @returns { task_id: 0xstring, execution_time: number }[]
    */
-  async getAutomationTimeMissedQueue(): Promise<MissedTask[]> {
+  async getAutomationTimeMissedQueue(): Promise<MissedTask[][]> {
     const polkadotApi = await this.getAPIClient()
-    const resultCodec = await polkadotApi.query['automationTime']['missedQueue']()
-    return resultCodec.toJSON() as unknown as MissedTask[]
+    const resultCodec = await polkadotApi.query['automationTime']['missedQueueV2']()
```

Comment:
should we keep another set of tests for the v1? or has that been fully deprecated / removed from storage?

Reply:
not removed from storage, but all of the extrinsics no longer point to the old storage. afaik, they are unreachable/unusable. you can see them in the chain state, but there's nothing in them and you can't put anything in them either